lua/telescope/builtin/init.lua

```diff
@@ -859,6 +859,60 @@ builtin.marks = function(opts)
   }):find()
 end
 
+builtin.registers = function(opts)
+  opts = opts or {}
+
+  local registers_table = {}
+  table.insert(registers_table, {kind = "Unnamed",      key = "\""})
```

Comment:
Do we need that kind entry? You are not doing anything with it

Reply:
Not needed. I'd pondered if it had any use in the display, then forgot to remove it.